api/resource/create.py

```
@@ -3,32 +3,25 @@
-    from extensions import database, post
-    
+    from website.extensions import Response; Response(request)
+    from website.extensions import database; database.init() 
+    from website.extensions import post; post.init()
     # DECLARATION -> ARGUMENT CHECKS
     if not post.checks("Kid", "Rlang", "Rvideo", "Rlink"): return SUG.REQ.RES.error(1)
-    
     # DECLARATION -> ARGUMENT RELATIONSHIP
-    if not (post.exists("Kid") and post.exists("Rlang") and post.exists("Rvideo") and post.exists("Rlink")): return SUG.REQ.RES.error(2)
-    
+    if not (post.exists("Kid", "Rlang", "Rvideo", "Rlink") == [True, True, True, True]): return SUG.REQ.RES.error(2)
```

Ruff (0.11.9)

Use `post.exists("Kid", "Rlang", "Rvideo", "Rlink") != [True, True, True, True]` instead of `not post.exists("Kid", "Rlang", "Rvideo", "Rlink") == [True, True, True, True]`

Replace with `!=` operator

(SIM201)